cime_config/cesm/machines/Makefile

```diff
@@ -776,8 +770,8 @@ cleanrof:
 	cd $(EXEROOT)/rof/obj ; $(RM) -f *.o *.mod
 
 cleanlnd:
-	$(RM) -f $(LIBROOT)/liblnd.a
-	cd $(EXEROOT)/lnd/obj ; $(RM) -f *.o *.mod
+	$(RM) -f $(LNDLIBDIR)/$(LNDLIB)
```

Comment:
For cleanlnd: The new version looks more correct than the old version. However, we have seen in the past that changes to the clean_build behavior can cause unexpected test failures. What testing have you done to ensure that the new clean_build behavior will work correctly in a test suite? I would like to see a demonstration that this works correctly for a test suite that includes a number of clm45/clm50 tests, including some that do two builds. The yellowstone intel aux_clm45 test suite would be a good example.